src/Error/Error.hs

```diff
@@ -19,13 +19,11 @@ data Error = Error {
                    }
    deriving (Eq)
 
--- Note: the error code functionality works, but only for type errors
--- the code below can be uncommented when it is introduced holistically
+-- Note: the error code functionality works, but only for type errors so they are not displayed
 instance Show Error where
-   show (Error (TE e) i p) = {-ec ++ "TE" ++ show i ++ "\n" ++-} "Type error in " ++ name ++ show e
```

Comment:
The unused `i` was a warning, so I just ripped the whole thing out. Not like it will be much work to put it back in.